lib/matplotlib/tests/tmp_test.py

```diff
@@ -0,0 +1,52 @@
+import re
```

Comment:
this file seems spurious?

Reply:
Already fixed, saw it a moment too late.